torchrl/objectives/a2c.py

```diff
@@ -32,6 +34,15 @@
     VTrace,
 )
 
+@tensorclass
```

Comment:
doens't it work if we make the base class a tensorclass?

Reply:
Yes, It doesn't work.